### ISSUE TITLE
Invalid JSON causes PRINT to fail

### ISSUE THREAD

**Issue by rmoff** [confluentinc/ksql#1473]
Note missing opening `{`: 

```
$ kafkacat -b localhost:9092 -t test_map -C
"a42","category":"foo","timestamp":"2018-06-21 10:04:57 BST","deviceID":42,"deviceProperties":[{"status":"false","foo":"bar"}]}
```

KSQL can't cope with this: 

```
ksql> print 'test_map' from beginning;
com.fasterxml.jackson.databind.node.TextNode cannot be cast to com.fasterxml.jackson.databind.node.ObjectNode
```

We should have a better error message, such as "Data is not recognisable as valid JSON, Avro, or Delimited: <raw message>"

**Comment by vcrfxia** [comment]
Closing as this is no longer an issue on the current master:
```
ksql> print 'test' from beginning;
Format:STRING
12/18/19 2:20:01 PM PST , NULL , "a42","category":"foo","timestamp":"2018-06-21 10:04:57 BST","deviceID":42,"deviceProperties":[{"status":"false","foo":"bar"}]}
^C
```
